aderyn_core/src/detect/high/block_timestamp_deadline.rs

```diff
@@ -116,55 +116,23 @@ impl IssueDetector for BlockTimestampDeadlineDetector {
 
 #[cfg(test)]
 mod block_timestamp_deadline_detector_tests {
-    use crate::detect::{
-        detector::{detector_test_helpers::load_contract, IssueDetector},
-        high::BlockTimestampDeadlineDetector,
-    };
+    use serial_test::serial;
 
-    #[test]
-    fn test_block_timestamp_deadline_uniswap_v2_detector() {
```

Comment:
Can you readd this test please?

We want to test both the UniswapV2 and UniswapV3 contracts